### PR TITLE
[ISSUE #3078]Refactor ThreadUtils

### LINKED ISSUE #3078: [Enhancement] Refactor ThreadUtils
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

 Refactor ThreadUtils

### Describe the solution you'd like

 Refactor ThreadUtils

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by mxsm**
Fixes #3078

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Refactor ThreadUtils*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java [hunk 1]
```diff
@@ -20,22 +20,45 @@
...
 
 public class ThreadUtils {
 
     private static volatile long currentPID = -1;
 
-    public static void randomSleep(int min, int max) throws Exception {
-        // nextInt is normally exclusive of the top value, so add 1 to make it inclusive
-        int random = ThreadLocalRandom.current().nextInt(min, max + 1);
-        Thread.sleep(random);
+    public static void randomSleep(long min, long max) {
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java:29]: Suggest to rename method will be better.

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java [hunk 2]
```diff
@@ -20,22 +20,45 @@
...
+        randomSleep(min, max, TimeUnit.MILLISECONDS);
+    }
 
+    public static void randomSleep(long min, long max, TimeUnit timeUnit) {
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java:33]: Suggest to rename method will be better.

File: eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java [hunk 3]
```diff
@@ -20,22 +20,45 @@
...
+        try {
+            long timeout = ThreadLocalRandom.current().nextLong(min, max + 1);
+            timeUnit.sleep(timeout);
+        } catch (InterruptedException ignore) {
+            //ignore
+        }
     }
 
-    public static void randomSleep(int max) throws Exception {
+    public static void randomSleep(long max) {
```
Comment (jonyangx) [inline eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java:43]: Suggest to rename method will be better.

Comment (mxsm) [reply eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/ThreadUtils.java:43]: > Suggest to rename method will be better.

@jonyangx Good idea, I will do and resubmit the pr later

**Review by jonyangx — APPROVED** [review]

**Pull request merged**